micro-benchmarks/matmul_relu.py

```diff
@@ -0,0 +1,84 @@
+import sre_compile
+import torch
+from benchmark_helper import time_with_torch_timer
+from torchinductor.triton_ops.matmul import matmul_out as triton_mm_out
+from torchinductor.triton_ops.matmul import matmul as triton_mm
+import torchdynamo
+import torchinductor.config as config
+
+config.triton.use_mm = True
+
+@torchdynamo.optimize("inductor", nopython=True)
+def inductor_mm(a, b):
+    return torch.mm(a, b)
+
+def torch_mm_relu(a, b):
+    return torch.nn.functional.relu(torch.matmul(a, b))
+
+def torch_mm(a, b):
```

Comment:
Above, `mm` was `mm`, but here its `matmul`. This is could lead to some confusion. there are nuanced differences, and they are not the same function.

Reply:
I see, thanks for noting that. I'll change them to all use `torch.mm`.